### PR TITLE
Raff/local testability

### PR DESCRIPTION
**Pull request by pauldria**

### CODE REVIEW

**Comment by pauldria** [comment]
I also updated the .gitignore to ignore stuff related to local testing.

File: Gemfile.lock [hunk 1]
```diff
@@ -0,0 +1,261 @@
+GEM
```
Comment (pauldria) [inline Gemfile.lock]: This is automatically generated when you run ``bundle exec jekyll serve`` so probably not absolutely necessary, but leaving it regardless.

**Comment by pauldria** [comment]
Closes #35

File: .gitignore [hunk 2]
```diff
@@ -1 +1,12 @@
 node_modules
+
+# Stuff to ignore related to local testing with Jekyll
+_site
+.sass-cache
+.jekyll-cache
+.jekyll-metadata
+vendor
+
+# Stuff to ignore related to emacs
```
Comment (lukasvermeer) [inline .gitignore]: I don't mind having this in the repo, but generally I would put this sort of IDE specific stuff in a global gitignore local to the machine.

File: local-development.md [hunk 3]
```diff
@@ -0,0 +1,35 @@
+# Testing locally
```
Comment (lukasvermeer) [inline local-development.md]: Do we want to add this file as an exclude in the _config.yml so it does not general a top level page on the site itself? Seems more like a GitHub thing.

Comment (pauldria) [reply local-development.md]: Yeah - I missed that on my testing, that it rendered as a page on the final site. :/ Doing one more PR.

Comment (lukasvermeer) [reply local-development.md]: Too late! I already merged it. :-)

**Pull request merged**